frontend/packages/topology/integration-tests/support/page-objects/topology-po.ts

```diff
@@ -35,6 +35,7 @@ export const topologyPO = {
     addNewAnnotations: '[data-test="add-button"]',
     deleteApplication: '[id="form-input-resourceName-field"]',
     connector: '[data-test-id="edge-handler"]',
+    routeDecorator: '[data-test="decorator-Open URL"]',
```

Comment:
Couldn't we reuse the aria-label here instead? Wdyt?

```suggestion
    routeDecorator: '[aria-label="Open URL"]',
```

Reply:
Updated based on the above comment

Reply:
Used data-id for better consistency